mtp_noms_ops/apps/security/templatetags/security.py

```diff
@@ -338,3 +339,8 @@ def extract_best_match(context, items):
         'item': best_match,
         'total_remaining': max(len(item_list)-1, 0),
     }
+
+
+@register.filter
+def get_human_readable_check_field(field):
+    return CHECK_DETAIL_RENDERED_MAPPING[field]
```

Comment:
it's safer to do `.get(field, field)` so that a new/unexpected field still renders _something_ vaguely readable on the page without error